anda/apps/helium-browser-bin/net.imput.helium.metainfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="runtime">
+  <id>net.imput.helium</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only AND BSD-3-Clause</project_license>
+
+  <name>Helium Browser</name>
+  <summary>Private, fast, and honest web browser based on Chromium.</summary>
+
+  <icon type="remote">
+    https://github.com/imputnet/helium/blob/main/resources/branding/app_icon/raw.png
+  </icon>
```

Comment:
The icon should be provided by the desktop shortcut, not remotely like this, appstream-generator does not know how to fetch this for some reason

Reply:
```suggestion

```